haystack/nodes/file_converter/pdf.py

```diff
@@ -61,6 +69,11 @@ def __init__(
 
         self.sort_by_position = sort_by_position
         self.multiprocessing = multiprocessing
+        self.ocr = ocr
```

Comment:
@silvanocerza Yes, “trusting” the user is never the ideal choice. I've done like this to follow the pattern of other pieces of code in Haystack. 

IMHO, it would be better to exchange in the whole codebase these Literal string choices by (str,Enum).